src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -49,16 +49,27 @@ export class DynamicFacetRange extends DynamicFacet implements IComponentBinding
       localizedString: () => l('To'),
       section: 'CommonOptions'
     }),
+    field: ComponentOptions.buildFieldOption({ required: true, section: 'CommonOptions' }),
```

Comment:
I need to add this here for postProcessing to work. I don't document it here because it's already documented in the DynamicFacet